ui/src/preview.tsx

```diff
@@ -0,0 +1,117 @@
+// Copyright 2020 H2O.ai, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { Model, S } from 'h2o-wave'
+import React from 'react'
+import { stylesheet } from 'typestyle'
+import { Component, XComponents } from './form'
+import { cards } from './layout'
+import { clas, cssVar, cssVarValue } from './theme'
+import { bond, wave } from './ui'
+import * as Fluent from '@fluentui/react'
+
+const css = stylesheet({
+  card: {
+    borderRadius: 4,
+  },
+  img: {
+    display: 'flex',
+    flexDirection: 'column',
+    justifyContent: 'space-between',
+    height: '100%',
```

Comment:
Is this needed?

Reply:
Not after removing the div for the image